src/Peering/Peering/Az.Peering.psd1

```diff
@@ -114,8 +121,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Update version of AutoMapper library used
-* Fix case sensitivity bug in ''Get-AzPeeringLocation'' for ''-Kind'' parameter'
+        ReleaseNotes = '* Preview Release for Microsoft Peering Service'
```

Comment:
Do not manually alter release notes information as this is automatic as part of our publishing process. This information comes from the changes in your `Changlog.md` file under the `Upcoming Release` header.